Content.Shared/EntityEffects/Effects/PlantMetabolism/RobustHarvest.cs

```diff
@@ -0,0 +1,21 @@
+using JetBrains.Annotations;
+using Robust.Shared.Prototypes;
+using Robust.Shared.Random;
+
+namespace Content.Shared.EntityEffects.Effects.PlantMetabolism;
+
+[UsedImplicitly]
+[DataDefinition]
```

Comment:
you have a few instances of UsedImplicitly and DataDefinition tho these are inherited from EntityEffect